Liquibase in dev mode detects duplicate changeLog file

**Describe the bug**
During startup of quarkus dev mode, Liquibase tries to parse the configured (yaml) file containing the changeLog, this fails due to 2 such files being found in the classpath.  This error does not occur in normal mode (running the runner jar).

**Expected behavior**
Liquibase only finds one changeLog file and parses it.

**Actual behavior**
The following exception is thrown, blocking further startup
```
2020-04-05 19:09:25,554 ERROR [io.qua.application] (main) Failed to start application: java.lang.IllegalStateException: liquibase.exception.ChangeLogParseException: Error parsing db/changeLog.yaml
        at io.quarkus.liquibase.runtime.LiquibaseRecorder.doStartActions(LiquibaseRecorder.java:75)
        at io.quarkus.deployment.steps.LiquibaseProcessor$configureRuntimeProperties41.deploy_0(LiquibaseProcessor$configureRuntimeProperties41.zig:98)
        at io.quarkus.deployment.steps.LiquibaseProcessor$configureRuntimeProperties41.deploy(LiquibaseProcessor$configureRuntimeProperties41.zig:36)
        at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:231)
        at io.quarkus.runtime.Application.start(Application.java:90)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at io.quarkus.runner.bootstrap.StartupActionImpl.run(StartupActionImpl.java:99)
        at io.quarkus.dev.IsolatedDevModeMain.firstStart(IsolatedDevModeMain.java:60)
        at io.quarkus.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:236)
        at io.quarkus.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:39)
        at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:131)
        at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:84)
        at io.quarkus.dev.DevModeMain.start(DevModeMain.java:113)
        at io.quarkus.dev.DevModeMain.main(DevModeMain.java:54)
Caused by: liquibase.exception.ChangeLogParseException: Error parsing db/changeLog.yaml
        at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:83)
        at liquibase.Liquibase.getDatabaseChangeLog(Liquibase.java:217)
        at liquibase.Liquibase.validate(Liquibase.java:1559)
        at io.quarkus.liquibase.runtime.LiquibaseRecorder.validate(LiquibaseRecorder.java:117)
        at io.quarkus.liquibase.runtime.LiquibaseRecorder.doStartActions(LiquibaseRecorder.java:59)
        ... 16 more
Caused by: java.io.IOException: Found 2 files that match db/changeLog.yaml
        at liquibase.util.StreamUtil.singleInputStream(StreamUtil.java:206)
        at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:26)
        ... 20 more
```

**To Reproduce**
Steps to reproduce the behavior:
1. Add the liquibase extension to your Quarkus project
2. Add a changeLog file to your resources
3. Startup in dev mode

**Configuration**
Relevant properties are:
```yaml
quarkus:
  liquibase:
    change-log: db/changeLog.yaml
    migrate-at-start: true
    validate-on-migrate: true
```
The change log file is custom, but it seems to be failing when I convert to db/changeLog.xml (the default) as well.

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Linux 4.15.0-94-generic #95-Ubuntu SMP Wed Mar 25 12:12:51 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: OpenJDK 64-Bit Server VM 18.9 (build 11.0.5+10, mixed mode)
 - GraalVM version (if different from Java): N/A
 - Quarkus version or git rev: 1.3.1-Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): Gradle 6.3

**Additional context**
Running in dev mode seems to duplicate the changeLog file in the build directory:
```bash
$ find . -name changeLog.yaml
./src/main/resources/db/changeLog.yaml
./build/classes/java/main/db/changeLog.yaml
./build/resources/main/db/changeLog.yaml
```
A clean + build results in the file being copied to only one location:
```bash
$ gw clean build
$ find . -name changeLog.yaml
./src/main/resources/db/changeLog.yaml
./build/resources/main/db/changeLog.yaml
```
Might be an indication of what's going wrong.

--- a/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
+++ b/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
@@ -316,16 +316,7 @@ public class QuarkusDev extends QuarkusTask {
                     extension.outputDirectory().getAbsolutePath(),
                     res);
             context.getModules().add(moduleInfo);
-
-            final String outputClassDirectory = extension.outputDirectory().getAbsolutePath();
-            final String outputResourcesDirectory = extension.outputConfigDirectory().getAbsolutePath();
             context.getClassesRoots().add(extension.outputDirectory().getAbsoluteFile());
-            if (!outputClassDirectory.equals(outputResourcesDirectory)) {
-                final File configDir = extension.outputConfigDirectory().getAbsoluteFile();
-                if (configDir.exists()) {
-                    context.getClassesRoots().add(configDir);
-                }
-            }
             context.setFrameworkClassesDir(wiringClassesDirectory.getAbsoluteFile());
             context.setCacheDir(new File(getBuildDir(), "transformer-cache").getAbsoluteFile());
 

